setup.py

```diff
@@ -85,13 +85,20 @@ def get_version(version_file):
     ],
     python_requires=">=3.6",
     # What does your project relate to?
-    keywords="skeleton",
+    keywords="Technical Payload Testing (TPT) Report",
```

Comment:
Since this is a list of keywords, I suggest:
```suggestion
    keywords="tpt technical payload testing report",
```